### PR TITLE
win condition wood_gnome can have several winners

### PR DESCRIPTION
**Pull request by SimonHeimberg**
In win condition wood_gnome, all players with maximum points win the game. (In case they have the same points at the end by chance.) The message still only lists one of them.
Teams are still not supported.

**Type of change**
partial Bugfix

**Related issue(s)**
improves #3129

### CODE REVIEW

File: data/scripting/win_conditions/wood_gnome.lua [hunk 1]
```diff
@@ -98,18 +98,20 @@ local r = {
...
+         local end_msgs = lost_game_over
+         if points[i][2] >= win_points then
+            end_msgs = won_game_over
+            win_lost = 1
+         end
+         privmsg = end_msgs.body
          privmsg = privmsg .. msg
-         points[i][1]:send_to_inbox(lost_game_over.title, privmsg)
-         wl.game.report_result(points[i][1], 0, make_extra_data(points[i][1], wc_descname, wc_version, {score=points[i][2]}))
+         points[i][1]:send_to_inbox(end_msg.title, privmsg)
```
Comment (hessenfarmer) [inline data/scripting/win_conditions/wood_gnome.lua]:
```suggestion
         points[i][1]:send_to_inbox(end_msgs.title, privmsg)
```

**Review by hessenfarmer — CHANGES_REQUESTED** [review]
one typo inline.
please fix the messages as well, as currently only the first entry of the sorted table will be mentioned in the message.  
I was not aware that we do not have a teams option for woodgnome so it would be good to fix that as well for this (but this is not mandatory for this PR)

**Comment by SimonHeimberg** [comment]
> please fix the messages as well, as currently only the first entry of the sorted table will be mentioned in the message. I was not aware that we do not have a teams option for woodgnome so it would be good to fix that as well for this (but this is not mandatory for this PR)

How do you suggest to to fix the message? Remove mentioning the winner since wood_gnome is the only win condition doing this?

**Comment by hessenfarmer** [comment]
skipping the message would be the easy fix, collecting all winning players in a table in another iteration of the points table and concatenating their names in the message (use of ngettext would be required) would be the cherry on the cake

**Review by hessenfarmer — APPROVED** [review]

**Comment by hessenfarmer** [comment]
I can live with that remaining TODO.  
So I will merge when CI is green, unless you tell me you will fix the TODO right here in this PR.    
How do you want to be credited (name, nickname, both)?

**Comment by SimonHeimberg** [comment]
> I can live with that remaining TODO. So I will merge when CI is green, unless you tell me you will fix the TODO right here in this PR. How do you want to be credited (name, nickname, both)?

only nickname aDiscoverer is fine

**Pull request merged**